src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1534,6 +1567,11 @@ void Convolution::execute(dnnl::stream strm) {
         IE_THROW() << "Can't execute Convolution node with name: " << getName() << ", because executor is not compiled";
     }
 
+    if (pendingConstWeightReorder) {
+        primArgs[DNNL_ARG_WEIGHTS] = prepareWeightMemory(DnnlExtensionUtils::makeDescriptor(execPtr->getWeightDesc()))->GetPrimitive();
+        pendingConstWeightReorder = false;
+    }
```

Comment:
What is the reason of putting this check to `execute` instead of `prepareParams()`?

Reply:
we cannot be 100% sure that the constant data is ready to access at `prepareParam()` stage, considering that there may be a constant folding subgraph between the constant input node and `this` Conv node, which are executed after `CreatePrimitives()`, check this code snippets:

```cpp
void Graph::InitGraph() {
  ...
  CreatePrimitives();      // prepareParam() is invoked
  ...
  ExecuteConstantNodesOnly(); // const-folding subgraph is executed
```

 and `prepareParam()` is called inside `Node::createPrimitive()`.

Reply:
Ok. Got it.